cylc/flow/scheduler.py

```diff
@@ -708,9 +706,6 @@ def _load_pool_from_point(self):
 
     def _load_pool_from_db(self):
         """Load task pool from DB, for a restart."""
-        if self.options.startcp:
-            self.config.start_point = TaskID.get_standardised_point(
-                self.options.startcp)
```

Comment:
This seems to be unnecessary as duplication of https://github.com/cylc/cylc-flow/blob/b366e13d93aaed34e11c764fbf1483557fb5da2a/cylc/flow/config.py#L669-L687

Reply:
Confirm looks reasonable.